frontend/src/css/modules/base.scss

```diff
@@ -82,13 +82,19 @@ a {
 ::-webkit-scrollbar {
```

Comment:
@aayusharora waiting for your reply here. I am merging this for now. Please leave a comment and Rishabh can incorporate that as a separate pull request.